### PR TITLE
This fixes Issue #297 

### PR DESCRIPTION
**Pull request by hurbes**

### CODE REVIEW

File: packages/stacked_generator/pubspec.yaml [hunk 1]
```diff
@@ -7,15 +7,18 @@ environment:
...
   source_gen: ^1.0.0
   analyzer: ^1.3.0
   path: ^1.6.4
   # logger: ^0.9.0
   recase: ^4.0.0-nullsafety.0
   stacked:
-    ^2.0.0
-    # path: ../stacked
+    # ^2.0.0
+    path: ../stacked
```
Comment (FilledStacks) [inline packages/stacked_generator/pubspec.yaml]: You have to change back to non-path dependency for me to release

**Review by FilledStacks — CHANGES_REQUESTED** [review]
Awesome man! Well done. Just a few general maintenance admin tasks that you missed out on.

1. Change back the path references in stacked_generator
2. Increase the version to 2.0.1
3. Add an entry into the changelog to for 2.0.1 - Adds ReactiveValue and ReactiveList for reactive functionality

File: packages/stacked_generator/pubspec.yaml (hunk 1 above)
Comment (hurbes) [reply packages/stacked_generator/pubspec.yaml]: Done. please check

**Review by FilledStacks — APPROVED** [review]
Awesome! I appreciate the fast work.

**Pull request merged**